core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -112,7 +135,11 @@ private[op] class OpMultiClassificationEvaluator
       log.warn("The dataset is empty. Returning empty metrics.")
       MultiClassificationMetrics(0.0, 0.0, 0.0, 0.0,
         MulticlassThresholdMetrics(Seq.empty, Seq.empty, Map.empty, Map.empty, Map.empty),
-        MultiClassificationMetricsTopK(Seq.empty, Seq.empty, Seq.empty, Seq.empty, Seq.empty))
+        MultiClassificationMetricsTopK(Seq.empty, Seq.empty, Seq.empty, Seq.empty, Seq.empty),
+        MulticlassConfusionMatrixTopKByThreshold($(topKCm), $(thresholds), Seq.empty),
```

Comment:
I'm not sure we want to use the same set of thresholds for the topK confusion matrix as we do for the existing threshold metrics since they're just 3 * (topN = 3) sets of counts per threshold, while this may end up being somewhere in the 10*10 - 30*30 range for each threshold.

I think we should have a separate set of thresholds for the confusion matrices, and be pretty sparing with them.